backend/channel_plugin/apps/channels/views.py

```diff
@@ -2,7 +2,7 @@
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from rest_framework.decorators import api_view
-from .serializers import ChannelSerializer
+from .seriaizers import ChannelSeriali
```

Comment:
why?

Reply:
Sorry. Mistake while trying to resolve online.